### DISCUSSION TITLE
dAISy 2+ dual-channel AIS Receiver support

### DISCUSSION THREAD

**Discussion by H4z3W4z** [jvde-github/AIS-catcher#138] [Q&A]
Hi Just trying to verify is the dAISy 2+ dual-channel AIS Receiver is support by AIS-catcher.

**Comment by jvde-github** [comment]
The daisy does its own decoding but you can use the program to forward the messages and display on the built-in map.

On the -e switch:
https://github.com/jvde-github/AIS-catcher/releases/tag/v0.49

And some older discussion:

https://github.com/jvde-github/AIS-catcher/discussions/42

**Reply by H4z3W4z** [reply]
Thanks, I did give that a try and seems to be working. However, I'm currently land-locked and wasn't sure if everything was functional. Will be near the coast in 10 days and hope to get some signals. Is there a way to generate demo data or to test comms with the daisy while not near targets?

**Reply by jvde-github** [reply]
Not that I am aware of but the Wegmatt folks might have some tips. I have seen this functionality for the dAISy Hat but don't own a 2+ so cannot tell. Sorry.

**Reply by H4z3W4z** [reply]
Thanks again. I'll let you know how my testing proceeds with the 2+. BTW I was pleasantly surprised with the compile/build process on Ubuntu!
